### DISCUSSION TITLE
[Feature Request] Player control bar only appears during mouseover

### DISCUSSION THREAD

**Discussion by discodancepant** [vimeo/player.js#777] [Ideas]
I would like a new option from the [videos>Embed Presets] menu. I'm a "pro" user and am often thwarted by the control bar which only disappears when the video is playing. My video graphics depend on having that space available. If you could allow an option to change the behavior of videos to a single "play" button when the video is loaded, and then have the control bar appear on a mouseover event, that would be a great faeture. I'm sure there are tons of others who want this feature as well. 

I'm well aware I can disable the control bar altogether, but I don't want that. I want the control bar to be available for the user ... but only when they are hovering their mouse over the video, intending to actually use the menu.

**Comment by lssamlyon** [comment]
Thanks for the feedback.  We don't currently have a mode for this today, so Im going to move this over to our discussions tab and add it to our backlog of feature requests internally.

**Comment by eballeste** [comment]
this is very very related to [my request](https://github.com/vimeo/player.js/issues/682) I did almost a year ago; the play button shows up whenever you control the video playback programmatically. an option to only have this behavior happen on mouseover is exactly the solution we need.

**Comment by virtualgadjo** [comment]
hi, any news about this feature?
i'm fighting with options to kind of hook it (autoplay 1 with pause on loaded, it works but the control bar stays visible in this case...)
have a nice day 
(edited for spelling issues...)

**Comment by weizhaa** [comment]
Still not seeing this function being publiced.
